tests/scripts/check-generated-files.sh

```diff
@@ -102,4 +102,5 @@ check scripts/generate_visualc_files.pl visualc/VS2010
 check scripts/generate_psa_constants.py programs/psa/psa_constant_names_generated.c
 check tests/scripts/generate_psa_wrappers.py tests/include/test/psa_test_wrappers.h tests/src/psa_test_wrappers.c
 check tests/scripts/generate_bignum_tests.py $(tests/scripts/generate_bignum_tests.py --list)
+check tests/scripts/generate_config_tests.py $(tests/scripts/generate_bignum_tests.py --list)
```

Comment:
Was this deliberate or copypasta? I don't think `generate_config_tests.py` has a `--list` option, but is the `generate_bignum_tests.py` list compatible?

Reply:
Woops, copypasta. Fixed.

All the `generate_xxx_tests.py` use [the same `main` function from `test_data_generation.py`](https://github.com/Mbed-TLS/mbedtls-framework/blob/main/scripts/mbedtls_framework/test_data_generation.py#L179) and they take the same command line options.